### ISSUE TITLE
Issues in a server environment

### ISSUE THREAD

**Issue by Vykori** [BluePsychoRanger/SkyBlock_Collection#10]
I've added 3 datapacks to my server, `skyvoid_additions_1_20`, `skyvoid_worldgen_portal_room_1_20`, and `skyvoid_island_standard_1_20`

I see the following errors in the console when launching the 1.20.1 server:
```
[10:35:11 ERROR]: Couldn't load tag gm4_trades:register_trades as it is missing following references: skyvoid_additions:versioning/register_trades (from file/skyvoid_additions_1_20)
[10:35:11 ERROR]: Couldn't load tag load:gm4_trades/enumerate as it is missing following references: gm4_trades-1.3:enumerate (from file/skyvoid_additions_1_20)
[10:35:11 ERROR]: Couldn't load tag load:skyvoid_worldgen/pre_load as it is missing following references: skyvoid_worldgen:v1.0/versioning/enumerate (from file/skyvoid_worldgen_portal_room_1_20)
[10:35:11 ERROR]: Couldn't load tag load:pre_load as it is missing following references: #load:skyvoid_worldgen/pre_load (from file/skyvoid_worldgen_portal_room_1_20)
[10:35:11 ERROR]: Couldn't load tag load:_private/init as it is missing following references: load:_private/init (from file/skyvoid_additions_1_20), load:_private/init (from file/skyvoid_island_standard_1_20), load:_private/init (from file/skyvoid_worldgen_portal_room_1_20)
[10:35:11 ERROR]: Couldn't load tag load:skyvoid_worldgen as it is missing following references: skyvoid_worldgen:v1.0/versioning/resolve_load (from file/skyvoid_worldgen_portal_room_1_20)
[10:35:11 ERROR]: Couldn't load tag load:skyvoid_standard_skyblock as it is missing following references: skyvoid_standard_skyblock:versioning/resolve_load (from file/skyvoid_island_standard_1_20)
[10:35:11 ERROR]: Couldn't load tag load:gm4_trades/resolve_load as it is missing following references: gm4_trades-1.3:resolve_load (from file/skyvoid_additions_1_20)
[10:35:11 ERROR]: Couldn't load tag load:gm4_trades as it is missing following references: #load:gm4_trades/enumerate (from file/skyvoid_additions_1_20), #load:gm4_trades/resolve_load (from file/skyvoid_additions_1_20)
[10:35:11 ERROR]: Couldn't load tag load:skyvoid_additions as it is missing following references: skyvoid_additions:versioning/load (from file/skyvoid_additions_1_20)
[10:35:11 ERROR]: Couldn't load tag load:load as it is missing following references: #load:skyvoid_additions (from file/skyvoid_additions_1_20), #load:skyvoid_standard_skyblock (from file/skyvoid_island_standard_1_20), #load:skyvoid_worldgen (from file/skyvoid_worldgen_portal_room_1_20)
[10:35:11 ERROR]: Couldn't load tag load:_private/load as it is missing following references: #load:_private/init (from file/skyvoid_additions_1_20), #load:_private/init (from file/skyvoid_island_standard_1_20), #load:_private/init (from file/skyvoid_worldgen_portal_room_1_20)
[10:35:11 ERROR]: Couldn't load tag gm4_trades:modify_external_trader as it is missing following references: gm4_trades-1.3:wandering_trader/resolve_modify_external (from file/skyvoid_additions_1_20)
[10:35:11 ERROR]: Couldn't load tag skyvoid_worldgen:initialize as it is missing following references: skyvoid_standard_skyblock:versioning/resolve_place_island (from file/skyvoid_island_standard_1_20), skyvoid_worldgen:v1.0/versioning/resolve_initialize (from file/skyvoid_worldgen_portal_room_1_20)
[10:35:11 ERROR]: Couldn't load tag minecraft:load as it is missing following references: #load:_private/load (from file/skyvoid_additions_1_20), #load:_private/load (from file/skyvoid_island_standard_1_20), #load:_private/load (from file/skyvoid_worldgen_portal_room_1_20)
```

When I run `datapack list` the datapacks do still show up, though.
```
[10:38:13 INFO]: There are 4 data pack(s) enabled: [vanilla (built-in)], [file/skyvoid_additions_1_20 (world)], [file/skyvoid_island_standard_1_20 (world)], [file/skyvoid_worldgen_portal_room_1_20 (world)]
[10:38:13 INFO]: There are no more data packs available
```
Is this expected behavior?

**Comment by BluePsychoRanger** [comment]
That is not expected behavior. How did you download and install the datapacks? The proper way would have been to navigate to the `build` folder, download the `.zip` files and drag those files into the `datapacks` folder. It seems that you might have unzipped the packs which isn't necessary, (but also shouldn't normally cause an issue). 

I've tested on a vanilla world and it seems to work fine. Is your server running something other than vanilla?

**Comment by Vykori** [comment]
ty for the reply and the help! not sure why it's not working on my server, then.

> It seems that you might have unzipped the packs which isn't necessary, (but also shouldn't normally cause an issue).  

Indeed, I was trying different potential fixes. Initially I was using the zips as-provided.

> How did you download and install the datapacks?  

First I got the world working correctly in singleplayer, then I copied the world folder over to the  server and saw these errors. So I tried deleting everything in the world folder except the datapacks subfolder and had the server re-generate the world, but I was still getting the same errors.

**Comment by BluePsychoRanger** [comment]
Sorry for the delayed response, [discord](https://discord.gg/mzWSZuGatd) would be a better form of communication since I check that more frequently.

I'm not sure what the issue is. It seems that you got it working in singleplayer, so there shouldn't be any different from a vanilla server. Maybe try redownloading (a new version has been released for 1.20-1.20.2) and trying again
